src/contracts/LANDRegistry.js

```diff
@@ -4,6 +4,42 @@ import { env } from '../env'
 
 /** LANDToken contract class */
 export class LANDRegistry extends Contract {
+  static decodeLandData(string = '') {
```

Comment:
The argument here is called `string` but on the encode section we use `data`. I get that in this case we're using a `string` and returning an object, but I think it's a bit confusing given the encodeLand*Data* and decodeLand*Data* names.

Maybe `data` here too and let the default value act as 'typing'? or dataString?

what do you think?